buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/gradlebuild/unittestandcompile/UnitTestAndCompilePlugin.kt

```diff
@@ -190,7 +190,9 @@ class UnitTestAndCompilePlugin : Plugin<Project> {
                     mapOf(
                         "org.gradle.test.maxParallelForks" to test.maxParallelForks,
                         "org.gradle.ci.agentCount" to 2,
-                        "org.gradle.ci.agentNum" to agentNum
+                        "org.gradle.ci.agentNum" to agentNum,
+                        "org.gradle.internal.http.connectionTimeout" to 60,
```

Comment:
:X: I don't think this will work as expected. These system properties are passed to the JVM which executes the tests.  I think that these won't have an effect on tests forking Gradle (including daemon tests), since Gradle there runs on a different JVM than the test JVM.

If you want to increase the timeout for the tests, I think you are better off at reconfiguring the Gradle executer instead of setting the system properties for the test JVM.

Reply:
Thanks! Really helpful!